charts/kuberpult/values.yaml

```diff
@@ -0,0 +1,51 @@
+# Default values for ..
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+git:
+  url: "/repository"  # git@github.com/.../...
```

Comment:
please add a comment, eg.
"the url should not have a default value. We only give it a value here, because `kind` in combination with `helm install` does not allow to set values (without crashing at teardown)"